### PR TITLE
Updates to make FaIR run with climate-assessment

### PR DESCRIPTION
**Pull request by chrisroadmap**
Ongoing plumbing to get the iiasa climate-assessment > openscm-runner > fair chain working. Phew.

- ~~Tests added~~  N/A
- ~~Documentation added~~  N/A
- [x] Example added (in the documentation, to an existing notebook, or in a new notebook)
- [x] Description in ``CHANGELOG.rst`` added (single line such as: ``(`#XX <https://github.com/openscm/openscm-runner/pull/XX>`_) Added feature which does something``)

### CODE REVIEW

File: notebooks/climate-assessment.ipynb [hunk 1]
```diff
@@ -0,0 +1,207 @@
+{
```
Comment (znicholls) [inline notebooks/climate-assessment.ipynb:1]: Which data is being used for this notebook?!

Comment (chrisroadmap) [reply notebooks/climate-assessment.ipynb:1]: Jarmo's test data from climate-assessment: I assume safe for public consumption?

Comment (znicholls) [reply notebooks/climate-assessment.ipynb:1]: @jkikstra can you confirm?

**Comment by znicholls** [comment]
Good to go once @jkikstra has confirmed

**Comment by znicholls** [comment]
Releasing v0.4.4 now (https://github.com/openscm/openscm-runner/actions/runs/359354424) if you want to use it @chrisroadmap

**Pull request merged**